test/intl402/BigInt/prototype/toLocaleString/builtin.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2012-2019 Mozilla Corporation; Igalia, S.L. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
```

Comment:
I got the following error
"No license found in: intl402/BigInt/prototype/toLocaleString/builtin"